src/location.rs

```diff
@@ -23,7 +69,8 @@ pub enum EndUser {
     Client {
         /// The EndUser PublicKey
         public_key: PublicKey,
-        /// A random hash that maps to a SocketAddr.
+        /// A hash of the EndUser signature over their socket address.
```

Comment:
Just for clarity, what do we gain from this? Is this step necessary?

Reply:
It's just to not sprinkle the socketaddr wide and about in the network.